src/main/java/com/google/sps/servlets/TasksServlet.java

```diff
@@ -52,6 +61,71 @@ public TasksServlet(
     this.tasksClientFactory = tasksClientFactory;
   }
 
+  private List<Task> getTasks(TasksClient tasksClient) throws IOException {
+    List<TaskList> taskLists = tasksClient.listTaskLists();
+    List<Task> tasks = new ArrayList<>();
+    for (TaskList taskList : taskLists) {
+      tasks.addAll(tasksClient.listTasks(taskList));
+    }
```

Comment:
Alternative: `taskLists.forEach(l -> tasks.addAll(taskClient.listTasks(l)))` (feel free to choose)

Reply:
The alternative is great but I can't seem to handle the IOException from calling `tasksClient.listTasks(l)` inside `.addAll`. I would have to surround it in a try catch which I think is more clutter than the for loop. Any thoughts?
```
    taskLists.forEach(taskList -> {
      try {
        tasks.addAll(tasksClient.listTasks(taskList));
      } catch (IOException e) {
        // TODO Auto-generated catch block
        e.printStackTrace();
      }
    });
```

Reply:
Oh, Java handling checked exceptions weirdly. You did well sticking with the for loop, then

Reply:
Hahah cool!

Reply:
`RESOLVED`